src/actplayer.cpp

```diff
@@ -4172,32 +4152,26 @@ void actPlayer(Entity* my)
 				}
 
 				selectedEntity[PLAYER_NUM] = entityClicked(&clickedOnGUI, false, PLAYER_NUM, clickType); // using objects
-				if ( !selectedEntity[PLAYER_NUM] && !clickedOnGUI )
+				if ( selectedEntity[PLAYER_NUM] && !clickedOnGUI )
 				{
 					if ( clickType == ENTITY_CLICK_USE )
 					{
 						// otherwise if we hold right click we'll keep trying this function, FPS will drop.
-						if ( (*inputPressedForPlayer(PLAYER_NUM, impulses[IN_USE])) || (inputs.bControllerInputPressed(PLAYER_NUM, INJOY_GAME_USE)) )
```

Comment:
To discuss removing selectedEntityGimpTimer

Reply:
Will fix